tempodb/encoding/vparquet2/block_traceql.go

```diff
@@ -999,7 +1000,7 @@ func createSpanIterator(makeIter makeIterFn, primaryIter parquetquery.Iterator,
 		switch cond.Attribute.Intrinsic {
 		case traceql.IntrinsicSpanID:
 
-			pred, err := createStringPredicate(cond.Op, cond.Operands)
+			pred, err := createBytesPredicate(cond.Op, cond.Operands, true)
```

Comment:
placeholder until https://github.com/grafana/tempo/pull/3667 is merged with the new predicates

Reply:
The new predicates will not have a bytes predicate. I would just write a new one in `pkg/parquetquery/predicates.go`.

Reply:
Since the string predicate actually uses bytes array, reusing the string predicate works well. Any concerns about this?

Reply:
nope. not if it does what we want